Sources/XcodeProj/Objects/Project/PBXProj.swift

```diff
@@ -92,6 +92,35 @@ public final class PBXProj: Decodable {
 
         try rootGroup()?.assignParentToChildren()
     }
+
+    // MARK: Static Methods
+
+    public static func createPBXProj(path: Path) throws -> PBXProj {
```

Comment:
this could possibly be another `init` method with different parameters but then you would have bunch of logic in the initializer, so I suppose we can keep it as-is.

Reply:
I am up for either option, whichever you prefer